fdb-relational-core/fdb-relational-core.gradle

```diff
@@ -47,11 +47,13 @@ dependencies {
     testImplementation(libs.bundles.test.runtime)
     testCompileOnly(libs.bundles.test.compileOnly)
 
+    testImplementation project(path: ':fdb-extensions', configuration: 'tests')
```

Comment:
I thought about putting the new class in a `testFixtures`, but I couldn't get that to work without putting all the classes that anyone already depends on via this approach in the testFixture, and thought that would be better as a followup.